ethereum/ethereum-reth/babel.yaml

```diff
@@ -0,0 +1,93 @@
+version: 0.0.1
+container_uri: docker-daemon://ethereum-reth:latest
+sku_code: ETH-RETH
+org_id: null
+description: Ethereum Reth node 
+visibility: public
+protocol_key: ethereum
+variants:
+  # The same protocol can have different implementations. This field let identify what implementation(s) has been used.
+  # Variant name may additional contain protocol network name e.g. "eth-mainnet".
+  # This value will be available in a Rhai runtime via `node_env().variant_key`.
+  - key: reth_mainnet_archive
+    min_cpu: 12
+    min_memory_mb: 32000
+    min_disk_gb: 3500
+    archive_pointers:
+      - pointer: !store_id ethereum-reth-mainnet-archive-v1
+      - pointer: !store_id ethereum-reth-mainnet-archive-v1
+        new_archive_properties: [ "tracing-api-on-value" ]
+  - key: reth_sepolia_archive
+    min_cpu: 6
+    min_memory_mb: 16000
+    min_disk_gb: 1000
+    archive_pointers:
+      - pointer: !store_id ethereum-reth-sepolia-archive-v1
+      - pointer: !store_id ethereum-reth-sepolia-archive-v1 
+        new_archive_properties: [ "tracing-api-on-value" ]
+  - key: reth_mainnet_full
+    min_cpu: 8
+    min_memory_mb: 16000
+    min_disk_gb: 1500
+    archive_pointers:
+      - pointer: !store_id ethereum-reth-mainnet-full-v1
+      - pointer: !combination_disallowed
+        new_archive_properties: [ "tracing-api-on-value" ]
+  - key: reth_sepolia_full
+    min_cpu: 4
+    min_memory_mb: 8000
+    min_disk_gb: 400
+    archive_pointers:
+      - pointer: !store_id ethereum-reth-sepolia-full-v1
+      - pointer: !combination_disallowed
+        new_archive_properties: [ "tracing-api-on-value" ]
+firewall_config:
+  default_in: deny
+  default_out: allow
+  rules:
+    - key: Allow incoming ports TCP
+      description: Allow TCP inbound traffic on port 80, 443
+      protocol: tcp
+      direction: in
+      action: allow
+      ips:
+        - ip: 0.0.0.0/0
+          name: allow all traffic
+      ports:
+        - port: 80
+          name: HTTP port
+        - port: 443
+          name: HTTPS port
+        - port: 3000
+          name: Grafana port
+    - key: Allow P2P ports TCP/UDP
+      description: Allow inbound traffic on P2P ports
+      protocol: both
+      direction: in
+      action: allow
+      ips:
+        - ip: 0.0.0.0/0
+          name: allow all traffic
+      ports:
+        - port: 30303
+          name: P2P port
+        - port: 42069
```

Comment:
Reth doesn't have torrent support

Reply:
Yeah, I kind of rushed through the reth image (or rather Windsurfed) in order to get it to build so I could put the CI/CD pipeline together. I'll go through the finetuning soon